Sources/Prepare.swift

```diff
@@ -50,7 +50,14 @@ public struct Prepare: Command {
 
             let hasPrepared = try database.hasPrepared(preparation)
             // only prepare the unprepared
-            guard !hasPrepared else { return }
+            guard !hasPrepared else { 
+                if let model = preparation as? Entity.Type {
```

Comment:
The `Model.database` property gets set when the preparation is sent to `database.prepare()`. In the case of this guard, we are returning early. So we need to set the property ourselves.

It might be better if `database.prepare()` just returned early and took care of setting the `.database` property on models, but right now it throws if the database has already been prepared for that entity.

If we set the `model.database` property outside of the guard, then it would get set twice in the case that the model has not been prepared yet.